chain/chain/src/store.rs

```diff
@@ -1522,7 +1522,7 @@ impl<'a> ChainStoreUpdate<'a> {
         self.sync_head = Some(t.clone());
     }
 
-    pub fn save_largest_target_height(&mut self, height: &BlockHeight) {
+    pub fn save_largest_target_height(&mut self, height: BlockHeight) {
         self.largest_target_height = Some(height.clone());
```

Comment:
It seems the clone is not needed here anymore

Reply:
Yes, good call. I caught another one of those I had missed as well after you pointed this one out. I guess `clippy` didn't catch them as redundant clones because it probably categorized them as "clone on copy" instead.

Fixed in https://github.com/nearprotocol/nearcore/pull/2634/commits/4ca95be9497fd71d243cf5addf98027c17959db8